src/Game.cpp

```diff
@@ -31,6 +33,13 @@ Game::~Game() {
 // TODO: Consider precedence of ticking i.e. should monsters be ticked before player, etc
 void Game::updateAll() {
 	float dt = GetFrameTime();
+
+	Goblin::timeToSpawn -= dt;
```

Comment:
maybe enemies can have a static method that does this logic, as its gonna have to be duplicated for the different enemies. Also for the more advanced enemies we can check global gametime like after 5 minutes we spawn wolves or something.

Reply:
Yea I tried to implement this in Enemy as a static method. But I can't get `this` for `Game` in the static method when `new` an Enemy.

Reply:
maybe just make it take a game reference in the function